.github/workflows/e2e_libvirt.yaml

```diff
@@ -51,6 +51,8 @@ jobs:
       - name: Read properties from versions.yaml
         run: |
           sudo snap install yq
+          echo "KBS_VERSION=$(yq -e '.git.kbs.reference' versions.yaml)" >> "$GITHUB_ENV"
+          echo "RUST_VERSION=$(yq -e '.tools.rust' versions.yaml)" >> "$GITHUB_ENV"
```

Comment:
Should we also add:
```
echo "KBS_REPO=$(yq -e '.git.kbs. url' versions.yaml)" >> "$GITHUB_ENV"
```
and then use that below rather than hard-code the url?

Reply:
Agreed.